talk/morelanguage/initialization.tex

```diff
@@ -0,0 +1,199 @@
+\subsection[Init]{Initialization}
+
+\begin{frame}[fragile]
+  \frametitle{Initialization}
+  \begin{block}{Initialization}
+    \begin{itemize}
+      \item \href{https://imgur.com/3wlxtI0}{Obligatory meme}
```

Comment:
I do get the concern about external links. I tried for an hour or so to embed the video (mp4) into the slide deck but failed, since flash support was removed from Adobe. That's why the link is external.

Apart from that, I really enjoy the joke! :D

But if it bothers you, I have no troubles removing it. The content is confusing at best.

Reply:
As mentioned, I would drop the whole slide to just say, "it's complicated, here is what you need to know". Not even sure it needs a dedicated slide, we may as well say it when presenting.

Reply:
There you go #230.